### DISCUSSION TITLE
Unable to search for a specific metadata field

### DISCUSSION THREAD

**Discussion by gbigger** [Security-Onion-Solutions/securityonion#7592] [Unsupported Versions] [answered]
How do I search for a specific metadata field.  These are from the cisco asa syslog, we need to be able to search on the syslog ID.  This json parameter specifically the "code": 302015 is what i need to search to find specific codes from the asa.  But no matter what i search for it does not return any results.

},
    "@version": "1",
    "event": {
      "severity": 6,
      "ingested": "2022-03-16T20:10:00.005541370Z",
      "code": 302015,
      "original": "%ASA-6-302015: Built outbound UDP connection 291419487 for Azure-PRIVATE-Out:10.2.30.55/64178 (10.2.30.55/64178) to Inside:10.60.200.24/53 (10.60.200.24/53)",
      "timezone": "+00:00",
      "kind": "event",
      "module": "cisco",
      "action": "firewall-rule",
      "category": [
        "network"
      ],
      "type": [
        "info"
      ],
      "dataset": "cisco.asa"

**Comment by dougburks** [comment, marked as answer]
Are you using the Cisco ASA filebeat module?

Have you tried searching for `code:302015`?

If that doesn't work, try an even more basic search like `302015` just to verify that it can find the log.

**Reply by gbigger** [reply]
just searching for the number worked, thank you :-)
